Client/index.ts

```diff
@@ -43,6 +43,11 @@ export class Client {
 		const connection = Connection.open(url, typeof token == "string" ? token : undefined)
 		return connection && new Client(connection, typeof token == "string" ? undefined : token)
 	}
+	assumedClient(orgCode: string) {
```

Comment:
perhaps better as a verb? assume()?

Reply:
the auth class already had a thing called assume() that overwrites the whole token etc, so i wanted to name it something different than that, idk